### PR TITLE
fix(core-database): map block to transaction in findById

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

The `findById` method called by the `transactions/{id}` API endpoint does not map the block to the transaction.

[untested]

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

File: packages/core-database/src/repositories/transactions-business-repository.ts [hunk 1]
```diff
@@ -59,7 +59,8 @@ export class TransactionsBusinessRepository implements Database.ITransactionsBus
     }
 
     public async findById(id: string) {
-        return this.databaseServiceProvider().connection.transactionsRepository.findById(id);
+        const rows = return this.databaseServiceProvider().connection.transactionsRepository.findById(id);
```
Comment (faustbrian) [inline packages/core-database/src/repositories/transactions-business-repository.ts]:
```suggestion
        const rows = await this.databaseServiceProvider().connection.transactionsRepository.findById(id);
```

Comment (dated) [reply packages/core-database/src/repositories/transactions-business-repository.ts]: Yea, thanks :D

**Comment by dated** [comment]
```
  ● Transactions Business Repository › findById › should invoke findById on db repository

    TypeError: Cannot read property 'get' of undefined
```

Will have a look once home.

**Comment by dated** [comment]
All good now @faustbrian

**Pull request merged**